webapp/components/interop-dashboard.js

```diff
@@ -864,7 +855,28 @@ class InteropDashboard extends PolymerElement {
     return columnNumber === sortColumn;
   }
 
+  adjustSortIconIndex(index) {
```

Comment:
Per comment above, instead of using a negative number, create a separate function for e.g. getFocusAreaColumnIcon().